web/src/components/tasks/business-formation/BusinessFormation.tsx

```diff
@@ -215,6 +216,7 @@ export const BusinessFormation = (props: Props): ReactElement => {
       </div>
     );
   }
+  const isForeign = userData?.profileData.businessPersona == "FOREIGN";
```

Comment:
we could use this variable in the if-statement above to simplify the code and not do the same check twice